vllm_ascend/attention/attention_mask.py

```diff
@@ -72,14 +74,6 @@ def get_attn_mask(self, max_seq_len: int, dtype: torch.dtype,
         return self.attn_mask_cache[:max_seq_len, :max_seq_len].contiguous(
         ).to(device, non_blocking=True)
 
-    def get_pooling_mask(self, device):
-        if self.pooling_mask is None:
-            # the compressed attention mask for npu_fusion_attention sparse mode 4
-            self.pooling_mask = torch.triu(torch.ones(
-                2048, 2048), diagonal=1).to(torch.bool).to(device,
-                                                           non_blocking=True)
-        return self.pooling_mask
-
```

Comment:
have you compared the performance of the compressed attention mask for npu_fusion_attention which is in shape of [2048,2048], and the attention mask you constructed which is in shape of [max_seq_len,max_seq_len]?